rkt/image_rm.go

```diff
@@ -76,24 +78,32 @@ func rmImages(s *store.Store, images []string) error {
 				stderr("rkt: error retrieving aci infos for image %q: %v", key, err)
 				continue
 			}
-			name = aciinfo.Name
-			keys = append(keys, key)
+			imageMap[key] = aciinfo.Name
+			imageCounter[key]++
 		}
+	}
 
-		for _, key := range keys {
-			if err = s.RemoveACI(key); err != nil {
-				if serr, ok := err.(*store.StoreRemovalError); ok {
-					staleErrors++
-					stderr("rkt: some files cannot be removed for image %q (%q): %v", key, name, serr)
-				} else {
-					stderr("rkt: error removing aci for image %q (%q): %v", key, name, err)
-					continue
-				}
+	// Ajust error count
```

Comment:
Typo in "Ajust", should be "Adjust". Also, could you elaborate a bit more about the adjustment? Something like "Subtract the error count for the duplicated image ids"?